lms/djangoapps/verified_track_content/models.py

```diff
@@ -39,30 +39,24 @@ def move_to_verified_cohort(sender, instance, **kwargs):  # pylint: disable=unus
             course = get_course_by_id(course_key)
             existing_manual_cohorts = get_course_cohorts(course, CourseCohort.MANUAL)
             if any(cohort.name == verified_cohort_name for cohort in existing_manual_cohorts):
-                # Verify that a single random cohort exists in the course. Note that calling this method will create
-                # a "Default Group" random cohort if no random cohorts exist yet.
+                # Get a random cohort to use as the default cohort (for audit learners).
+                # Note that calling this method will create a "Default Group" random cohort if no random
+                # cohort yet exist.
                 random_cohort = get_random_cohort(course_key)
-                if not is_default_cohort(random_cohort):
```

Comment:
Should we also update the other usage of `is_default_cohort` to allow for multiple random cohorts? According to my grep, it's only used in this file and one other.
- https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/course_groups/cohorts.py#L433-L434 Update error string to say "at least one"
- https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/course_groups/cohorts.py#L452-L458 Get all random cohorts, check `len >= 1`, and `name in ` instead of `name ==`

At any rate, the import in line 15 of this file can be removed.